api/spec/spec_helper.rb

```diff
@@ -64,6 +64,12 @@
     Spree::Api::Config[:requires_authentication] = true
   end
 
+  config.include ActiveJob::TestHelper
+  config.after(:each) do
+    ActiveJob::Base.queue_adapter.enqueued_jobs = []
+    ActiveJob::Base.queue_adapter.performed_jobs = []
```

Comment:
Just curious - So these don't happen automatically when you use the `:test` adapter?  And there aren't any nicer helper methods?